### PR TITLE
Update for zig 0.12.0

### PR DESCRIPTION
**Pull request by rockorager**
- **libinput(ClickMethod): s/click_finger/clickfinger**
- **build: update to zig 0.12.0**

This PR updates requisite code for 0.12.0. I also changed `click_finger` to
`clickfinger` to match the libinput API and allow tests to pass.

### CODE REVIEW

File: src/types.zig [hunk 1]
```diff
@@ -365,7 +365,7 @@ pub const ScrollMethod = enum {
 
 pub const ClickMethod = enum {
     button_areas,
-    click_finger,
```
Comment (JayceFayne) [inline src/types.zig:368]: Good catch!

File: build.zig.zon [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.{
+    .name = "swayipc",
+    .version = "0.0.0",
```
Comment (JayceFayne) [inline build.zig.zon]: Idk, version 0.0.0 just seems wrong, maybe 0.2.0?

File: build.zig [hunk 3]
```diff
@@ -14,7 +16,8 @@ pub fn build(b: *std.Build) void {
             .optimize = optimize,
         });
 
-        exe.addModule("swayipc", swayipc);
+        exe.root_module.addImport("swayipc", swayipc);
+        // exe.addModule("swayipc", swayipc);
```
Comment (JayceFayne) [inline build.zig]: Any particular reason why this is a comment?

**Review by JayceFayne — CHANGES_REQUESTED** [review]
Thanks for the PR! I added some suggestions

File: build.zig.zon (hunk 2 above)
Comment (rockorager) [reply build.zig.zon]: Heh, works for me!

File: build.zig (hunk 3 above)
Comment (rockorager) [reply build.zig]: Just an oversight...I commented it out when updating the `addImport` call and forgot to delete it.

**Comment by rockorager** [comment]
Updated to v0.2.0 and removed the commented out line.

**Comment by JayceFayne** [comment]
Thanks!

**Pull request merged**